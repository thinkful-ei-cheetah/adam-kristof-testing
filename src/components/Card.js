import React from 'react';
import '../card.css'

const Card=({title, content})=>{
  console.log(title)
  return <div className="Card">
    <h3>{title}</h3>
    <p>{content}</p>
  </div>
}

export default Card;