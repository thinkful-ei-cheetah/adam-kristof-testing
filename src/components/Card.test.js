import React from 'react';
import ReactDOM from 'react-dom';
import renderer from  'react-test-renderer';
import Card from './Card';

describe('Card Tests', ()=>{
  it('Renderes to the page', ()=>{
    const div = document.createElement('div');
    ReactDOM.render(<Card/>, div);
    ReactDOM.unmountComponentAtNode(div)
  })

  it('Renders title and content correctly', ()=>{
    const tree = renderer
    .create(<Card title={'The Title'} content={'THis is content'} />)
    .toJSON()
    expect(tree).toMatchSnapshot();
  })
})