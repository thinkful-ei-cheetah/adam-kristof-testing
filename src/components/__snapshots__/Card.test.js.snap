// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Tests Renders title and content correctly 1`] = `
<div
  className="Card"
>
  <h3>
    The Title
  </h3>
  <p>
    THis is content
  </p>
</div>
`;
